addons/web_tour/static/src/js/tour_step_utils.js

```diff
@@ -157,5 +157,43 @@ return Class.extend({
             },
         ].map(this.addDebugHelp(this._getHelpMessage('mobileKanbanSearchMany2X', modalTitle, valueSearched)));
     },
+    /**
+     * Utility steps to save a form and wait for the save to complete
+     *
+     * @param extra_trigger additional save-condition selector
+     */
+    saveForm(extra_trigger) {
+        return [{
+            content: "save form",
+            trigger: '.o_form_button_save:contains("Save")',
+            extra_trigger,
+            run: 'click',
+            auto: true,
+        }, {
+            content: "wait for save completion",
+            trigger: '.o_form_button_edit',
+            run() {},
+            auto: true,
+        }];
+    },
+    /**
+     * Utility steps to cancel a form creation or edition.
+     *
+     * Supports creation/edition from either a form or a list view (so checks
+     * for both states).
+     */
+    exitForm() {
```

Comment:
May I suggest to rename this to `cancelForm` or `discardForm` ? I came here from a tour because I wanted to know whether this function was meant to exist the form by saving or by discarding. When you arrive here it gets obvious because the other function is `saveForm`, but while you're in a tour it's not that obvious.